packages/suite-web/e2e/tests/analytics/events.test.ts

```diff
@@ -0,0 +1,192 @@
+                expect(deviceConnectEvent.totalDevices).to.equal('1');
+                expect(deviceConnectEvent.language).to.equal('en-US');
+                expect(deviceConnectEvent.model).to.equal('T');
+            });
+
+        // transport-type
+        cy.wrap(requests)
+            .its(2)
+            .should(transportTypeEvent => {
+                expect(transportTypeEvent.c_type).to.equal(EventType.TransportType);
+                expect(transportTypeEvent.type).to.equal('BridgeTransport');
+                expect(parseInt(transportTypeEvent.version, 10)).to.not.equal(NaN);
+            });
+
+        // device-disconnect is logged 4th
+        cy.task('stopEmu');
+        cy.wrap(requests).should('have.length', 4);
+        cy.wrap(requests).its(3).should('have.property', 'c_type', EventType.DeviceDisconnect);
+    });
+
+    it('reports suite-ready after enabling analytics on app initial run', () => {
+        cy.task('startEmu', { wipe: true });
+        cy.task('setupEmu', {
+            needs_backup: false,
+            mnemonic: 'all all all all all all all all all all all all',
+        });
+        cy.task('startBridge');
+
+        cy.intercept({ hostname: 'data.trezor.io', url: '/suite/log/**' }, req => {
+            const params = urlSearchParams(req.url);
+            requests.push(params);
+        });
+
+        cy.prefixedVisit('/');
+
+        // change few settings to see if it is different from default values in suite-ready
+        cy.getTestElement('@suite/menu/settings').click();
+
+        // change language
+        cy.getTestElement('@settings/language-select/input').click();
+        cy.getTestElement('@settings/language-select/option/cs').click();
+
+        // change fiat
+        cy.getTestElement('@settings/fiat-select/input').click();
+        cy.getTestElement('@settings/fiat-select/option/czk').click();
+
+        // change BTC units
+        cy.getTestElement('@settings/btc-units-select/input').click();
+        cy.getTestElement('@settings/btc-units-select/option/Satoshis').click(); // sat
+
+        // change dark mode
+        cy.getTestElement('@theme/color-scheme-select/input').click();
+        cy.getTestElement('@theme/color-scheme-select/option/dark').click();
+
+        // disable btc, enable ethereum and goerli
+        cy.getTestElement('@settings/menu/wallet').click();
+        cy.getTestElement('@settings/wallet/network/btc').click();
+        cy.getTestElement('@settings/wallet/network/eth').click();
+        cy.getTestElement('@settings/wallet/network/tgor').click();
+
+        // custom eth backend
+        cy.getTestElement('@settings/wallet/network/eth/advance').click();
+        cy.getTestElement('@settings/advance/select-type/input').click();
+        cy.getTestElement('@settings/advance/select-type/option/blockbook').click();
+        cy.getTestElement('@settings/advance/url').type('https://eth.marek.pl/');
```

Comment:
Sounds like a nice domain for ETH node @marekrjpolak

Reply:
⠀⠀⠀⠀⠀⠀⣴⣿⡆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⣿⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⣿⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⣿⣿⡇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⣠⣦⡄⣿⣿⡇⢠⣶⡄⢀⣤⡀
⢀⣠⡄⣿⣿⣧⣿⣿⣧⣿⣿⣧⣼⣿⣿
⣾⣿⡇⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⣿⣿⣧⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿
⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡟
⠈⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠟⠁
⠀⠀⠀⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇